### ISSUE TITLE
The application pop-up / option bar doesn't appear in Fanbox

### ISSUE THREAD

**Issue by bempfoobq052021** [leoding86/webextension-pixiv-toolkit#268]
tl;dr There's no option to download from Fanbox images for some reason (it's still there a few days ago, I think, which is a few days after manifest v3 update)

![スクリーンショット 2024-08-21 170058](https://github.com/user-attachments/assets/3d7545d6-d1f2-4383-aef2-37c08b96f22c)

**The bug**
The option bar at the bottom of my screen is now gone.

**Expected behavior**
There should be a small option bar near the bottom of my screen. 

**Example work from Pixiv**
It works in Pixiv, but not in Fanbox.
![スクリーンショット 2024-08-21 170735](https://github.com/user-attachments/assets/3d2e1e8b-70c9-4f76-8566-f1463accf360)

**Browser and Version (please complete the following information):**
Google Chrome Ver. 127.0.6533.120 (Official Build) (64 bit)

**Settings of extension**
Language: English
Activate download panel automatically : Yes
Download panel position :  Center
Download panel style : Type 1
Show reload in extension popup page : Yes
Here's a screenshot of my settings.
![スクリーンショット 2024-08-21 170855](https://github.com/user-attachments/assets/fbe838c1-caa4-4bba-8dcc-38db160cbe68)

**Additional context**
I've been downloading from Pixiv after manifest v.3 update so I didn't notice the change in Fanbox. There's an addiional system error though: I can't change the Fanbox Download Task Settings because the option reverts to the default after I close or reload the page.

**Comment by cyxceven** [comment]
I have the same issue. No download bar at the bottom of the screen for Fanbox. It still works on Pixiv.
Using Microsoft Edge Version 128.0.2739.67 (Official build) (64-bit)

**Comment by leoding86** [comment]
I can't reproduce the issue. If you can install unpack extension manually, you can try the latest version in [release](https://github.com/leoding86/webextension-pixiv-toolkit/releases/tag/6.4.0) page.

**Comment by Fraetor** [comment]
I have this issue with v5.5.3 on Firefox. Does this extension still support installing on Firefox? I couldn't get the linked version to install, with an error: `background.service_worker is disabled.`

**Comment by Fraetor** [comment]
![image](https://github.com/user-attachments/assets/c13159f5-7444-4506-84de-12b73d03a201)

Looking at the dev tools, it is failing to parse some JSON somewhere.

**Comment by kajunbowser** [comment]
Noticing this issue on v4.6.2 on Brave v1.87.192 (Mar 4, 2026). Looks like a CORS policy has gotten in the way.

<img width="773" height="576" alt="Image" src="https://github.com/user-attachments/assets/078d12a0-3fb3-491a-a3ac-2c933718eb89" />

**Comment by bempfoobq052021** [comment]
Oh my, sorry for the very late reply. It's been resolved for me for some times ago. The remedy was simply to refresh the page a few times - sometimes 6 times or more, but the button will show up. I got the same CORS error. I think maybe fanbox uses two domain name for their services, one that calls on the api.fanbox.cc (which probably trigger the cors warning, hard to fix tbh) and the other that uses normal fanbox.cc _Proobably_ 
I haven't had that much problem lately because I can just refresh a few times. Thanks for the very useful extension btw!

EDIT 13/03/2026: Added the CORS issue screenshot

<img width="700" height="261" alt="Image" src="https://github.com/user-attachments/assets/5c5ace2b-d371-40cb-a627-77038bbbd0a4" />

**Comment by bempfoobq052021** [comment]
Can I close this or do I wait?
